src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -432,11 +432,11 @@ private void AddNameAndTypeArgumentsOrParameters(INamedTypeSymbol symbol)
             }
         }
 
-        private ImmutableArray<ImmutableArray<CustomModifier>> GetTypeArgumentsModifiers(NamedTypeSymbol underlyingTypeSymbol)
+        private ImmutableArray<ImmutableArray<CustomModifier>> GetTypeArgumentsModifiers(NamedTypeSymbol? underlyingTypeSymbol)
         {
             if (this.format.CompilerInternalOptions.IncludesOption(SymbolDisplayCompilerInternalOptions.IncludeCustomModifiers))
             {
-                if ((object)underlyingTypeSymbol != null)
+                if ((object?)underlyingTypeSymbol != null)
```

Comment:
```suggestion
                if (underlyingTypeSymbol is not null)
```

Reply:
➡️ I left this for a separate refactoring.